# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/ComputedStyle.h
# Commit: 2ba84268f8ac
# Full Hash: 2ba84268f8ac2baf36477e62bc03ade97966d865
# Author: Eitan Isaacson <eitan@monotonous.org>
# Date: 2025-09-30 17:19 +0000
# Regressor Bug: 1987945
# File Overlap Count: 1
# Overlapping Files: accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Bug 1987945 - P3: Support details relation in anchored targets in remote accessibles. r=Jamie,layout-anchor-positioning-reviewers,layout-reviewers,emilio,firefox-style-system-reviewers
# ==============================================================================

diff -r 7aaaa4514127 -r 2ba84268f8ac layout/style/ComputedStyle.h
--- a/layout/style/ComputedStyle.h	Tue Sep 30 17:19:31 2025 +0000
+++ b/layout/style/ComputedStyle.h	Tue Sep 30 17:19:31 2025 +0000
@@ -196,6 +196,8 @@
 
   bool HasAnchorPosReference() const;
 
+  bool MaybeAnchorPosReferencesDiffer(const ComputedStyle* aOther) const;
+
   ComputedStyle* GetCachedInheritingAnonBoxStyle(
       PseudoStyleType aPseudoType) const {
     MOZ_ASSERT(PseudoStyle::IsInheritingAnonBox(aPseudoType));
